roles/openshift_ansible_inventory/tasks/main.yml

```diff
@@ -24,6 +24,13 @@
     owner: root
     group: libra_ops
 
+# This cron uses the above location to call its job
+- name: Cron to keep cache fresh
+  cron:
+    name: 'multi_ec2_inventory'
+    minute: '*/10'
+    job: '/usr/share/ansible/inventory/multi_ec2.py --refresh-cache &> /dev/null'
```

Comment:
I'm not sure that everyone is going to want this cron job. I'd prefer it if this was a boolean, that's not set by default. Something like this:

```
when: oo_cron_refresh_cache
```